spec/cypress/app_commands/scenarios/admin/school_participants.rb

```diff
@@ -53,7 +55,8 @@
                                           date_of_birth: Date.parse("10/12/1982"),
                                           nino: "NI123456",
                                           teacher_reference_number: "9780824",
-                                          school_urn: school.urn
+                                          school_urn: school.urn,
+                                          npq_course: npq_course
 
   NPQ::CreateOrUpdateProfile.new(npq_validation_data: npq_validation_data).call
```

Comment:
I guess it will be an issue when I merge my PR into develop because CreateOrUpdateProfile got removed

Reply:
Maybe, will fix it when you merge your PR

Reply:
It's been merged now